test/F3B_test.go

```diff
@@ -0,0 +1,351 @@
+package integration
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	_ "net/http/pprof"
+
+	"github.com/stretchr/testify/require"
+	accessContract "go.dedis.ch/dela/contracts/access"
+	"go.dedis.ch/dela/core/txn"
+	"go.dedis.ch/dela/core/txn/signed"
+	"go.dedis.ch/dela/crypto"
+	"go.dedis.ch/dela/crypto/bls"
+	"go.dedis.ch/dela/crypto/ed25519"
+	"go.dedis.ch/dela/crypto/loader"
+	"go.dedis.ch/dela/dkg"
+	"go.dedis.ch/dela/dkg/pedersen"
+	"go.dedis.ch/dela/dkg/pedersen/types"
+	"go.dedis.ch/dela/internal/testing/fake"
+
+	"go.dedis.ch/dela/mino"
+
+	"go.dedis.ch/dela/mino/minoch"
+	"go.dedis.ch/dela/mino/minogrpc"
+	"go.dedis.ch/dela/mino/router/tree"
+
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/suites"
+	"go.dedis.ch/kyber/v3/xof/keccak"
+)
+
+func init() {
+	rand.Seed(0)
+}
+
+func Test_F3B(t *testing.T) {
```

Comment:
Remove F3B references and rename appropriately

Reply:
The F3B test has been removed. It didn't make sense to keep it.